openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -340,3 +340,10 @@ def pre_requisite_courses(self):
         Returns a list of ID strings for this course's prerequisite courses.
         """
         return json.loads(self._pre_requisite_courses_json)
+
+    @classmethod
+    def get_all_course_keys(cls):
+        """
+        Returns all course keys from course overviews.
+        """
+        return [course_overview.id for course_overview in cls.objects.all()]
```

Comment:
I'm almost positive that I tried using `values()`, but for some reason it didn't work.  I'll try it again and see what the issue was (if there was one).